packages/places/README.md

```diff
@@ -0,0 +1,215 @@
+# React Native AEP Places Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepplaces.svg)](https://www.npmjs.com/package/@adobe/react-native-aepplaces)
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepplaces)](https://www.npmjs.com/package/@adobe/react-native-aepplaces)
+
+`@adobe/react-native-aepplaces` is a wrapper around the iOS and Android [Adobe Experience Platform Places Extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-places) to allow for integration with React Native applications. Functionality to enable Adobe Places is provided entirely through JavaScript documented below.
```

Comment:
Functionality to enable Adobe Places is provided entirely through JavaScript documented below. (Should it be TypeScript?  Or just remove this sentence?

Reply:
I think its okay, since typescript is just a wrapper around javascript, and what they are actually invoking is already compiled to javascript for them. So a javascript app can use this library with no problem.